config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   root 'reviews#show'
-  resources :reviews, only: [:show, :create]
+  resources :reviews,       only: [:show, :create]
   resources :cards
+  resources :users,         only: [:new, :create, :edit, :update]
+  resources :user_sessions, only: [:new, :create, :destroy]
+
+  get '/sign_up', to: 'users#new', as: 'sign_up'
+  get '/log_in', to: 'user_sessions#new', as: :log_in
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.